src/ducks/balance/Balance.styl

```diff
@@ -95,3 +96,7 @@
     .Balance__bank
         maxed-flex-basis 60%
 
+    .Balance__history
+        margin-left -1rem
+        margin-right -1rem
+        margin-top -1rem
```

Comment:
What is those -1rem ? Is it to cancel the padding of the container ? I understand why this is done here but I think it is not clear that it is coupled to the Layout of Cozy-UI.

I think regrouping the two declarations  would make it clear that they are working on the same "feature" (cancelling the paddings of the Layout container).

```
.Balance__history
    margin-left -2rem
    margin-right -2rem

    +small-screen()
        &
	        margin-left -1rem
	        margin-right -1rem
	        margin-top -1rem
```

Then I wonder if it could be something shipped in Cozy-UI.

```
.Balance__history
  @extends $Layout__fullwidth
```

Reply:
Actually I found [this solution](https://css-tricks.com/full-width-containers-limited-width-parents/), but it doesn't work on the large layout with the fixed sidebar on the left. Maybe cozy-ui can ship something I will look at it.

Reply:
Ah, I knew there was something for this pattern. Check src/styles/mixins.styl ;)